### PR TITLE
UI tweaks

### LINKED ISSUE #463: UI Cosmetic Fixes
Per [Figma](https://www.figma.com/file/LkNVe5ob1LjGpbGU74Hcrs/Fuji---Design-Files-(DUP)?node-id=5463-82195&t=lwLa395a6IDUaOpb-0)

- [x] Increase logo size
- [x] Reduce lending screen's height
- [x] Update notification icons
- [x] Update notification links
- [x] Update notification styling
- [x] Use Inter font where needed (see Figma file) and make sure there's no FOUT
- [x] Update font sizes per Figma file

### LINKED ISSUE #464: UI issues
- [x] I'm connected to Optimism, and I'm on the markets page. I click on the Gnosis vault and I'm redirected to the borrow page. I can see the active vault is the one on Gnosis, but both selected chains are Optimism. That's pretty bad because it blocks the user to proceed. If I input both collateral and debt, the button will remain disabled. 
Here, the borrow chain has to be Gnosis as I clicked on the Gnosis vault.
image

- [x] We removed the name of the lending provider and the chain for each step, but it's a bit difficult to spot the differences between the different routes. I suggest we put back those details so it looks more explicit. The deposit step will become "Deposit WETH to Agave on [icon] Gnosis" and the borrow step: "Borrow USDC from Agave on [icon] Gnosis".
image

- [x] I'm connected to Optimism, and I land on the borrow page. Both chains on the form are Polygon while they have to be Optimsim.

### PR DESCRIPTION
**Pull request by ferostar**
close #463 
close #464

### CODE REVIEW

**Comment by ferostar** [comment]
Hey @NikolaiYurchenko  I started this one but couldn’t finish today so you go ahead now please!

- Finish the notification styling first. You can see I simplified the hell out of the thing, now all notifications are custom so we only need to modify it once. Main things missing are… height/width, top margin when no link… and polishing it once it's all done.
- Updating fonts, everywhere
- Some styling (step padding in tx modal, etc)

File: packages/frontend/styles/globals.css [hunk 1]
```diff
@@ -1,9 +1,11 @@
-/* @font-face {
-  font-family: "Inter";
+@font-face {
+  font-family: 'Inter';
+  src: url('../public/assets/fonts/Inter-Regular.woff2') format('woff2'),
```
[the remark this answers is not in the document]
Comment (NikolaiYurchenko) [reply packages/frontend/styles/globals.css:3]: Haven't heard of it till now. It is interesting, let me try it

[the remark this answers is not in the document]
Comment (NikolaiYurchenko) [reply packages/frontend/styles/globals.css]: @ferostar good catch! seems like we can get rid of fonts completely

**Comment by ferostar** [comment]
@NikolaiYurchenko not working on Firefox

Captura de pantalla 2023-04-18 a la(s) 11 46 25

**Comment by NikolaiYurchenko** [comment]
> not working on Firefox

Tried, seems ok. Maybe try to refresh page without cache?

**Comment by brozorec** [comment]
> > Tried, seems ok. Maybe try to refresh page without cache? @ferostar
> 
> Still :/. @brozorec can you give it a try?

When I open it in a private navigation, it worked. But not in a normal navigation ... Cleared the cache for https://v2-staging-465-dot-fuji-306908.ey.r.appspot.com/ and still same result which is weird.

**Review by brozorec — APPROVED** [review]
Approving this PR, we may fix the tablet issues in another PR.

**Pull request merged**